src/main/java/software/amazon/cloudformation/resource/ResourceTypeSchema.java

```diff
@@ -166,6 +166,10 @@ public ResourceTypeSchema(Schema schema) {
                     taggingValue.setCloudFormationSystemTags(Boolean.parseBoolean(value.toString()));
                 } else if (key.equals(ResourceTagging.TAG_PROPERTY)) {
                     taggingValue.setTagProperty(new JSONPointer(value.toString()));
+                } else if (key.equals(ResourceTagging.TAG_PERMISSIONS)) {
+                    HashSet<String> tagPermissions = new HashSet<>();
+                    ((List<?>) value).forEach(p -> tagPermissions.add(p.toString()));
+                    taggingValue.setTagPermissions(tagPermissions);
```

Comment:
do we need to make it `HashSet`? Why dont we use list, then you dont really need to convert it

Reply:
I follow the type definition in `Handler.java` as `Set<String>` and I think the handler permissions is also set as an `Hashset`, but I am not sure whether it is a must for tagging permissions. I can change it to list.